core/domain/stats_services.py

```diff
@@ -875,6 +877,36 @@ def create_stats_model(exploration_stats: stats_domain.ExplorationStats) -> str:
     return instance_id
 
 
+def create_stats_model(
```

Comment:
In what circumstances should this be used rather than create_and_put?

Reply:
I've removed "create_and_put" in the latest updates. So this is comment is obselete.